RunningOrder/Shared/Components/Search/SearchBarSuggestions.swift

```diff
@@ -30,29 +34,38 @@ extension SearchBarSuggestions {
 
         var body: some View {
             ScrollView {
+//                Picker("Search Scope", selection: $logic.searchScope) {
+//                    ForEach(Scope.allCases, id: \.self) { scope in
+//                        Text(scope.title)
+//                            .tag(scope)
+//                    }
+//                }
+//                .labelsHidden()
+//                .pickerStyle(SegmentedPickerStyle())
+//                .padding()
```

Comment:
is it still useful to keep that commented code ?

Reply:
yes, it is bugged for now, but it could interesting in the futur. so I added this as a comment to get it later, as the code was not here previously